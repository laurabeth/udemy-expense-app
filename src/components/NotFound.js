import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => (
  <div>
    404 - <Link to="/"> Return to Expensify Home</Link>
  </div>
);

export default NotFound;
