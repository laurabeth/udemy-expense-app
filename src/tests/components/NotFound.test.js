/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/NotFound";

describe("not found", () => {
  it("renders", () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
