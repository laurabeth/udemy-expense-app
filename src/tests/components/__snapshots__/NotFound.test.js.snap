// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found renders 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
     Return to Expensify Home
  </Link>
</div>
`;
